modules/db/variables.tf

```diff
@@ -1,9 +1,36 @@
-variable "app-name" {}
+variable "app_name" {}
 
-variable "id_of_vpc" {}
+variable "vpc_id" {}
 
 variable "master_password" {}
 
 variable "vpc_rds_security_group_ids" {}
 
-variable "both_db_subnets_name" {}
\ No newline at end of file
+variable "db_subnets_name" {}
+
+variable "availability_zones" {
+  description = "A list of availability zones in which the subnets will be created"
+  type        = list(any)
+}
+
+variable "master_username" {
+  description = "Username for the database"
+  type        = string
+}
+
+variable "engine" {
+  description = "The engine that the database runs on"
+  type        = string
+  default     = "aurora"
+}
+
+variable "database_name" {
+  description = "The engine that the database runs on"
```

Comment:
looks like the description has been copy/pasted ;)

Reply:
Done.